src/features/announcements/Announcement.tsx

```diff
@@ -0,0 +1,84 @@
+import React, { useContext } from "react";
+import { Button } from "components/ui/Button";
+import { Context } from "features/game/GameProvider";
+import { announcements } from "features/announcements";
+import fence from "assets/land/goblin_fence.png";
+import { useShowScrollbar } from "lib/utils/hooks/useShowScrollbar";
+import classNames from "classnames";
+
+const CONTENT_HEIGHT = 400;
+export const Announcement: React.FC = () => {
+  const { ref: itemContainerRef, showScrollbar } =
+    useShowScrollbar(CONTENT_HEIGHT);
+  const { gameService } = useContext(Context);
+
+  const storedDate = localStorage.getItem("announcementLastRead");
+
+  function onAcknowledge() {
+    gameService.send("ACKNOWLEDGE");
+  }
+
+  const filtered = storedDate
+    ? announcements.filter(
+        (announcement) => announcement.date > new Date(storedDate)
+      )
+    : announcements;
+
+  // Sort latest first
+  filtered.sort((a, b) => b.date.getTime() - a.date.getTime());
+
+  return (
+    filtered && (
+      <div
+        ref={itemContainerRef}
+        style={{ maxHeight: CONTENT_HEIGHT }}
+        className={classNames("overflow-y-auto", {
+          scrollable: showScrollbar,
+        })}
+      >
```

Comment:
<img width="97" alt="Screen Shot 2022-06-12 at 9 15 19 am" src="https://user-images.githubusercontent.com/11745561/173207898-9f52cb9c-7e76-403a-9a19-66d0e1558623.png">

For UI consistency consider adding the 'X' button. This is the pattern that most of our modals follow.

We don't want to add too much friction for players that don't want to read the announcements. The purpose of this feature is just to give them another avenue of finding out about features - hence a "X" to quickly close would be ideal